Cartfile.resolved

```diff
@@ -1,3 +1,3 @@
-github "Quick/Nimble" "v8.0.5"
-github "Quick/Quick" "v2.2.0"
-github "box/box-ios-sdk" "v3.1.0"
+github "Quick/Nimble" "v9.2.0"
```

Comment:
Is there a reason you updated Quick and Nimble dev dependencies?

Reply:
None in particular - let me revert